src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_header">
```

Comment:
The class name 'moyo_header' could be more semantic. Consider using 'header' or another meaningful name that describes the role of the element.